.azure/pipelines/helix-matrix.yml

```diff
@@ -48,3 +52,41 @@ jobs:
     - name: Helix_logs
       path: artifacts/log/
       publishOnError: true
+      
+# Helix ARM64
+- ${{ if ne(variables['System.TeamProject'], 'internal') }}:
```

Comment:
Why do this only for public runs e.g. are you dealing w/ another missing internal Helix queue❔ aspnetcore-helix-matrix-internal is disabled in any case.